python/sidecar/http.py

```diff
@@ -31,54 +20,54 @@ async def sidecar_index():
 
 
 @sidecar_api.post("/ingest")
-async def upload() -> IngestResponse:
-    # response = sidecar.ingest.run_ingest()
+async def http_ingest() -> IngestResponse:
```

Comment:
Why rename this one?

Reply:
To match the sidecar command name and url (/ingest) like you have for most of the commands.
Also, IMO we won’t have upload available via the sidecar api. Right? 
And I think that you will implement a different file/url/code for the upload feature.

I will also revert this change.

Reply:
Given our Slack huddle, I'm ok if we keep this one named `ingest`